lib/App.js

```diff
@@ -1833,27 +1827,113 @@ class App {
             return true;
           }
         }
-      ]);
+      ]); 
 
-      // Create a custom object to inject en flag for the placeholder.
-      flowArguments.push({
-        type: argumentType,
-        name: argumentName,
-        placeholder: { en: argumentPlaceholder },
-      });
+      flowCardArgument.type = argumentType;
+      flowCardArgument.name = argumentName;
+
+      if (['text', 'autocomplete', 'number', 'date', 'time', 'device'].includes(flowCardArgument.type)) {
+        const { argumentPlaceholder } = await inquirer.prompt([
+          {
+            type: 'input',
+            name: 'argumentPlaceholder',
+            message: 'Enter the placeholder for the argument',
+            validate: input => input.length > 0,
+          }
+        ]);
+
+        flowCardArgument.placeholder = { en: argumentPlaceholder };
+      } 
+
+      if (flowCardArgument.type === 'number' || flowCardArgument.type === 'range') {
+        const { argumentMin, argumentMax, argumentStep } = await inquirer.prompt([
+          {
+            type: 'input',
+            name: 'argumentMin',
+            message: 'What is the minimum value?',
+            default: 0,
+            validate: input => !Number.isNaN(Number(input))
+          },
+          {
+            type: 'input',
+            name: 'argumentMax',
+            message: 'What is the maximum value?',
+            default: 100,
+            validate: input => !Number.isNaN(Number(input))
+          },
+          {
+            type: 'input',
+            name: 'argumentStep',
+            message: 'What is the step value?',
+            default: 1,
+            validate: input => !Number.isNaN(Number(input))
+          },
+        ]);
+
+        flowCardArgument.min = argumentMin;
+        flowCardArgument.max = argumentMax;
+        flowCardArgument.step = argumentStep;
+      }
+
+      if (flowCardArgument.type === 'dropdown') {
+        let { confirmDropdownValues } = await inquirer.prompt([
+          {
+            type: 'confirm',
+            name: 'confirmDropdownValues',
+            message: 'Do you want to add values to your dropdown?',
+            default: false
+          }
+        ]);
+
+        let dropdownValues = [];
```

Comment:
You're not changing `dropdownValues`, only pushing to it, so this can be `const`.

Doesn't VSCode/ESLint change this automatically?